### DISCUSSION TITLE
How do we debug databend?

### DISCUSSION THREAD

**Discussion by Weijun-H** [databendlabs/databend#6917] [Q&A] [answered]
Hi everyone,

I am new to rust and databend. How do we debug in databend? Now I tried to use `GDB` to connect to the databend process like mysql. Is there a better one to use? Thank you.

**Comment by Xuanwo** [comment]
Most of the time, I debug will `print` :rofl: 

Start databend `RUST_LOG=debug` can print leggings to `stderr`.

**Comment by RinChanNOWWW** [comment, marked as answer]
I use VSCode with plugins(Rust Analyzer + CodeLLDB).
![image](https://user-images.githubusercontent.com/33975039/182006597-9e8eb1e8-a065-4f31-ba5e-f4aef0c36f30.png)
